### PR TITLE
Add JMX ports for DC and SW

### PR DESCRIPTION
**Pull request by velvia**

### CODE REVIEW

**Comment by znep** [comment]
Can you add the SERVICE_*_NAME settings as listed in https://github.com/socrata-platform/region-coder/pull/2/files#r30274233 so the apps-marathon names can work both for JMX and for access to the service?

File: coordinator/docker/ship.d/run [hunk 1]
```diff
@@ -7,6 +7,13 @@ exec su socrata -c '/usr/bin/java \
     -Xmx${JAVA_XMX} \
     -Xms${JAVA_XMX} \
     -Dconfig.file=${SERVER_CONFIG} \
+    -Djava.net.preferIPv4Stack=true \
+    -Dcom.sun.management.jmxremote \
```
Comment (znep) [inline coordinator/docker/ship.d/run]: Do you actually need this line?

Comment (velvia) [inline coordinator/docker/ship.d/run]: Apparently not.  Will remove it.  Thanks.

**Comment by velvia** [comment]
@znep updated. 
Also would you mind reminding me where the SERIVCE_*_NAME settings are again?  Thx.

**Pull request merged**